docs/cms_config.ini

```diff
@@ -39,6 +39,7 @@ login = getting_started/index.html
 register = getting_started/index.html
 content_libraries = course_components/libraries.html
 content_groups = course_features/cohorts/cohorted_courseware.html
+enrollment_tracks = course_features/cohorts/cohorted_courseware.html
```

Comment:
I meant to add a comment here.  This was the location @sstack22 pointed me to initially. She said there is work in Docs to update this section. We planned to change this in the future. 

FYI @sstack22 @catong